### PR TITLE
Feature/feedback connection

### PR DESCRIPTION
**Pull request by MMSaavedraC**
work for https://trello.com/c/8kluZAzM/163-feedback-connection

### CODE REVIEW

File: consultantmarket/views.py [hunk 1]
```diff
@@ -172,19 +174,55 @@ def getURL(request):
...
+        msg = EmailMultiAlternatives(subject='Skills Searcher Feedback')
+        if form.is_valid():
+            attachment = form.cleaned_data['attachment']
+            try:
+                msg.attach(filename=attachment.name, content=attachment.read())
+            except:
                 pass
-        msg.send()
+            adminMsg = administratorEmail(form, request, msg)
+            adminMsg.send()
```
Comment (mmartinezintersys) [inline consultantmarket/views.py]: I think we should validate if this admin email is sent succesfully similar to what it's done for user email in line 194, and if its sent successfully then continue with user email. And maybe think on a way to display one Success or Error message.

One example to see an error is having invalid configuration values for the application email (like an invalid password) and try to send the feedback. Then you will see the application breaks when calling this function.

File: consultantmarket/views.py [hunk 2]
```diff
@@ -172,19 +174,55 @@ def getURL(request):
...
+            sent = usrMsg.send()
+            if sent:
+                print('sent succesfully')
+                messages.success(request,"The feedback was sent successfully")
+                return HttpResponseRedirect('/feedback/')
+        else:
+            textForm = sendText(request.POST)
+            if textForm.is_valid():
+                adminMsg = administratorEmail(textForm, request,msg)
+                adminMsg.send()
```
Comment (mmartinezintersys) [inline consultantmarket/views.py]: Same as in line 191

File: consultantmarket/views.py [hunk 3]
```diff
@@ -172,19 +174,55 @@ def getURL(request):
...
+    if request.method == 'POST':
+        form = sendForm(request.POST, request.FILES)
+        
+        msg = EmailMultiAlternatives(subject='Skills Searcher Feedback')
+        if form.is_valid():
+            attachment = form.cleaned_data['attachment']
+            try:
+                msg.attach(filename=attachment.name, content=attachment.read())
+            except:
                 pass
```
Comment (mmartinezintersys) [inline consultantmarket/views.py]: There should be something to handle this exception. Maybe returning an error message; it can be reviewed on the standup

**Comment by MMSaavedraC** [comment]
Hi @mmartinezintersys your comments are addressed, please review again.

**Pull request closed without merging**